ambry-commons/src/main/java/com/github/ambry/commons/LoggingNotificationSystem.java

```diff
@@ -69,7 +70,16 @@ public void onBlobUndeleted(String blobId, String serviceId, Account account, Co
         ", " + serviceId + ", accountName " + (account == null ? null : account.getName()) + ", accountId" + (
             account == null ? null : account.getId()) + ", containerName " + (container == null ? null
             : container.getName()) + ", containerId " + (container == null ? null : container.getId()));
+  }
 
+  @Override
+  public void onBlobReplicated(String blobId, String serviceId, Account account, Container container,
+      DataNodeId sourceHost) {
+    logger.debug("onBlobReplicated {}", blobId,
```

Comment:
this string format is super weird. Could you please fix it, just like the string format in onBlobCreated. And BTW, we should probably fix the onBlobDeleted and onBlobUndeleted.

Reply:
Sure. Fixed the code format of these functions.